src/app/_css/fonts.css

```diff
@@ -0,0 +1,31 @@
+@font-face {
+  font-family: 'Acta Bold';
+  src: url('/fonts/Acta-Bold.woff2') format('woff2'),
+  url('/fonts/Acta-Bold.woff') format('woff');
+  font-weight: normal;
```

Comment:
I haven't used font-face in a while, but I _think_ you can declare both Acta versions as `font-family: 'Acta'`, but with `font-weight: bold` for the bold version. Otherwise you're declaring 2 different fonts, instead of 2 versions of 1 font

Reply:
a-ha! that makes sense, i'll look into it.
same would apply for Colfax i assume? 
but that means i'll have to define a variable for the weight as well then, correct?

Reply:
i've updated it to have `--font-weight` vars instead and use just one of each. looks cleaner indeed.